spvc/src/main.cc

```diff
@@ -325,7 +326,7 @@ int main(int argc, char** argv) {
   options.SetMSLDiscreteDescriptorSets(msl_discrete_descriptor);
 
   shaderc_spvc::CompilationResult result;
-  shaderc_spvc_status status = shaderc_spvc_status_configuration_error;
```

Comment:
I wonder why it complained here.
Seems harmless.
But fine with me.

Reply:
It was pointing out that due to the if/elseif/else block below the value being stored here will never be used. This is a minor nit I think, but it was correct in pointing it out.